services/google_scraping_service/job_enqueuer.go

```diff
@@ -0,0 +1,38 @@
+package google_scraping_service
+
+import (
+	"errors"
+	"log"
+
+	"github.com/gocraft/work"
+	"github.com/gutakk/go-google-scraper/db"
+	"github.com/gutakk/go-google-scraper/models"
+)
+
+const (
+	invalidKeyword = "invalid keyword"
+)
+
+var EnqueueScrapingJob = func(savedKeyword models.Keyword) error {
+	if len(savedKeyword.Keyword) == 0 {
+		return errors.New(invalidKeyword)
+	}
+
+	enqueuer := work.NewEnqueuer("go-google-scraper", db.GetRedisPool())
+
+	job, err := enqueuer.Enqueue(
+		"scraping",
```

Comment:
Again as an improvement to remove all `-ing`, how about renaming to "search"? It would map nicely with the new name of `google_search_service`.

Reply:
Fixed in de9d39e0563cefb62d222ae4e9bcd9e4ea35aeb6